R/utils.R

```diff
@@ -1,3 +1,7 @@
+#' @importFrom utils globalVariables
+utils::globalVariables(c("pval_nominal", 'perc', 'gene_id',  'type', 'eQTL',
+                         'chr', 'gene_mid', "maf", "data", "vcf", "gff", "dna"))
+
```

Comment:
What are these variable used for? I'd prefer to avoid any global variables if we can.

Reply:
I did this to remove the warnings that look like: 

`no visible binding for global variable ‘maf’`

If you have a preferred solution, I'm happy to change it!

Reply:
Ugh, I know that warning well, it's the worst. Do you know where that is coming from? There are a few solutions depending on the cause. If it's not obvious I can probably track it down.

Reply:
Three (vcf, gff, dna) I removed because they were related to the vcfR package. The others are all data.frame column names.

Reply:
I think when the column already exists using `$` is fine but it complains if you use it to create a new column. You should be able to fix it by using `df[, "col"]` or `df[["col"]]` instead of `df$col`.

Reply:
Cool! Fixed :)